src/components/cc-orga-member-list/cc-orga-member-list.stories.js

```diff
@@ -1,4 +1,4 @@
-import './cc-orga-member-list.js';
+import './cc-orga-member-list.smart.js';
```

Comment:
this is actually not necessary because the `injectAuthForSmartComponentsPlugin` already does the import.
This will be removed anyway.
TODO @florian-sanders-cc